kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java

```diff
@@ -216,7 +221,26 @@ public boolean createPattern() {
                                     .acceptability(ACCEPTABLE))
             );
         }
+        boolean isSuccess = composer.commitSession(session);
+        setPropertyValue(PATTERN, pattern);
+        stampViewModel.setValue(TIME, getPatternLastStampTime());
+        return isSuccess;
+    }
+
+    public String getPatternTitle() {
+        return ((EntityFacade) getPropertyValue(PATTERN)).description();
+    }
+
+    public Long getPatternLastStampTime() {
+        EntityFacade patternFacade = getPropertyValue(PATTERN);
+        Entity pattern = EntityService.get().getEntityFast(patternFacade.nid());
+        IntIdSet idSet = pattern.stampNids();
+        OptionalLong latestStamp = idSet.intStream().mapToLong(i -> ((StampEntity)EntityService.get().getEntityFast(i)).time()).max();
```

Comment:
Can we use `getViewProperties().stampCalculator()`?

Reply:
updated to use the StampCalculator along with the code that @rbradley813 suggested to get the STAMP.